src/payment/strategies/stripev3/stripev3-payment-strategy.spec.ts

```diff
@@ -360,6 +360,28 @@ describe('StripeV3PaymentStrategy', () => {
                     expect(response).toBe(store.getState());
                 });
 
+                it('with credit card passing back the client token', async () => {
```

Comment:
🍹 I know the rest of the file is written like this, probably a pattern we have in other places, but this line should be "readable". _it with credit card passing back the client token_ makes little to no sense 😅 

I guess we are missing the _what_ in that phrase: _it submits(?) with a credit card and passes back the client token_?